pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -63,8 +64,8 @@ def configure_ddp(self, model, device_ids):
         self._ddp_kwargs["find_unused_parameters"] = self._ddp_kwargs.get(
             "find_unused_parameters", True
```

Comment:
i'm not sure why this is the default. this incurs a perf hit and is different from the DDP default

Reply:
It was added in this PR: #4382, there must be a good reason, @SeanNaren?

Reply:
Hey @awaelchli. Not sure Will looked into the default for `find_unused_parameters`. Let's stick to Pytorch default which is False right ?

Reply:
It should be false, it is only recommended to do this if necessary: https://pytorch.org/docs/stable/notes/ddp.html#internal-design

Reply:
ok, made it default False to be in line with pytorch DDP.

Reply:
this could be a nice speedup for distributed training jobs. @SeanNaren n00b question: is there a way to verify this on the lightning benchmarks?